token.go

```diff
@@ -40,7 +40,7 @@ func loadToken(args *GlobalOptions, host string) (string, error) {
 
 func ensureConfigPathExists(configDir string) error {
 	dotConfigNtgrrc := dotConfigDirName(configDir)
-	err := os.MkdirAll(dotConfigNtgrrc, os.ModeDir)
+	err := os.MkdirAll(dotConfigNtgrrc, os.ModeDir|0700)
```

Comment:
Built a version of `ntgrrc` for Windows and Linux (amd64+aarch64) with this modification (spacing removal applied by `go fmt`); confirmed working as expected

Reply:
perfect, thanks for the feedback